docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -199,52 +206,102 @@ Whereas in production, it would return:
 <img src="https://bigcommerce.customcdn.net/img/image.jpg" />
 ```
 
-As highlighted above, the helper is configured to rewrite *local* URLs to a `assets/cdn/` subfolder. The `stencil bundle` command will exclude this local `assets/cdn/` subfolder from the bundle that it creates. This filtering circumvents the 50 MB size limit on the resulting .zip file.
+As highlighted above, the helper is configured to rewrite *local* URLs to an `assets/cdn/` subfolder. The `stencil bundle` command will exclude this local `assets/cdn/` subfolder from the bundle that it creates. This filtering circumvents the 50 MB size limit on the resulting ZIP file.
 
 ### {{money}}
 
-Format number length, thousands delimiter, and decimal delimiter.
+```handlebars
+{{money value n s c}}
+```
+Formats number length, thousands delimiter, and decimal delimiter.
 
-[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/money.js).
+#### Parameters
+
+- `value` {Number}: The number to format.
+- `n` {Integer}: Length of decimal.
+- `s` {Mixed}: Thousands delimiter.
+- `c` {Mixed}: Decimal delimiter.
+
+#### Example
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=money).
+```handlebars
+{{money 1000 2 ',' '.'}}
+```
+
+[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/money.js).
 
 ### {{getFontLoaderConfig}}
 
-Returns font loader config as JSON string.
+```handlebars
+{{getFontLoaderConfig filepath}}
+```
+Returns font-loader configuration as a JSON string.
+
+#### Parameters
 
-[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getFontLoaderConfig.js).
+- `filepath` {String}: Filepath to the config file.
+
+#### Example
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getFontLoaderConfig).
+```handlebars
+{{getFontLoaderConfig './lib/fonts'}}
+```
+
+[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getFontLoaderConfig.js).
 
 ### {{getFontsCollection}}
 
+```handlebars
+{{getFontsCollection}}
+```
 Returns `<link>` elements for configured fonts.
 
-[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getFontsCollection.js).
+#### Example
+
+```handlebars
+{{getFontsCollection}}
+<!-- <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine"> -->

```

Comment:
@junedkazi Can you please review this example?

Reply:
Something like the one below based on all the fonts referenced in the theme
```<link href="https://fonts.googleapis.com/css?family=Open+Sans:,400italic,700|Karla:700|Lora:400|Volkron:|Droid:400,700|Crimson+Text:400,700&display=swap" rel="stylesheet">
```